pages/k8s/supported-versions.md

```diff
@@ -30,24 +30,13 @@ juju deploy charmed-kubernetes
 ```
 
 It is also possible to deploy a specific version of the bundle by including the
-revision number. For example, to deploy the **Charmed Kubernetes** bundle for the Kubernetes 1.21
+revision number. For example, to deploy the **Charmed Kubernetes** bundle for the Kubernetes 1.23
 release, you could run:
 
 ```bash
-juju deploy charmed-kubernetes --revision=733
+juju deploy cs:charmed-kubernetes-862
```

Comment:
we want cs: here because we're explicitly talking about an old release; CH will use the CS shim to do the right thing when it encounters a cs:-prefixed artifact.